src/Aquifer.Jobs/Services/SendGridEmailService.cs

```diff
@@ -0,0 +1,112 @@
+using Aquifer.Common.Clients;
+using Aquifer.Common.Services;
+using SendGrid;
+using SendGrid.Helpers.Mail;
+using EmailAddress = Aquifer.Common.Services.EmailAddress;
+
+namespace Aquifer.Jobs.Services;
+
+public class SendGridEmailService : IEmailService
+{
+    private readonly SendGridClient _sendGridClient;
+
+    // TODO move this to DI and don't make a network call in a constructor
```

Comment:
I didn't change this pre-existing strategy of fetching a key vault secret in the constructor in this PR but it looks like we could refactor to do this as part of DI instead.